app/components/org-group-form.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+
+import FormMixin from 'open-event-frontend/mixins/form';
+//OrgGroupForm
```

Comment:
Codacy found an issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6000753)